build.gradle

```diff
@@ -36,6 +36,7 @@ configurations {
 dependencies {
     implementation("org.springframework.boot:spring-boot-starter-web")
     implementation("org.springframework.boot:spring-boot-starter-thymeleaf")
+    implementation("org.thymeleaf.extras:thymeleaf-extras-java8time:3.0.1.RELEASE")
```

Comment:
I think it is already included

Reply:
Yes. Probably in `spring-boot-starter-thymeleaf` ?